mailbox/pom.xml

```diff
@@ -60,6 +60,7 @@
         <module>plugin/quota-search-elasticsearch</module>
         <module>plugin/quota-search-scanning</module>
         <module>plugin/spamassassin</module>
+        <module>tools</module>
```

Comment:
there's already a `tool` module, what's the difference ?

Reply:
I split the `tool` module into `tools/*` modules. So no difference and one is the evolution of the other one.

Later in that PR I drop `tool`.

Because each 'tool' have a specific set of dependencies and I did not want that to be mixed together anymore.